index.html

```diff
@@ -2463,7 +2463,7 @@ <h3>Data Schemas</h3>
 more data schemas that provide <a>verifiers</a> with enough information to
 determine if the provided data conforms to the provided schema. Each
 <code>credentialSchema</code> MUST specify its <code>type</code> (for example,
-<code>JsonSchemaValidator2018</code>), and an <code>id</code> <a>property</a>
+<code><a href="https://w3c-ccg.github.io/vc-json-schemas/">CredentialSchema2022</a></code>), and an <code>id</code> <a>property</a>
```

Comment:
```suggestion
<code>CredentialSchema2022</code>), and an <code>id</code> <a>property</a>
```

This link is likely to break over the years, so the suggestion is to not link to it. It's somewhat unavoidable to link to it non-normatively in the appendix... with the assumption that the bibliography will be updated over time, and eventually will go into the ReSpec bibliography database, which can then point people to a newer spec if the link dies.

Either that, or wait until there is a stable link at W3C for CredentialSchema2022 and then point it that.

Reply:
Agree, will update this.

Reply:
removed href